client/blocks/reader-feed-header/style.scss

```diff
@@ -67,9 +67,13 @@
 		max-height: 16px * 5.9;
```

Comment:
Let's remove this max-height too, as it causes issues on smaller screens

<img width="461" alt="image" src="https://user-images.githubusercontent.com/1699996/194131938-ec9c4080-a219-457d-a83b-c2b1aea0f7f3.png">

Other than that, looks good to me.

Reply:
Good call! Removed in https://github.com/Automattic/wp-calypso/pull/68653/commits/7f67eb9e24e2f4980bf367cdfaeb6988e139f420